### PR TITLE
DEVDOCS-3803-update-payment-docs

### PR DESCRIPTION
**Pull request by traci707**
# [DEVDOCS-3803](https://jira.bigcommerce.com/browse/DEVDOCS-3803)

## What changed?
Updated payment docs

### CODE REVIEW

File: reference/payment_processing.yml [hunk 1]
```diff
@@ -911,6 +926,18 @@ components:
...
+                    test_mode: false
+                    type: gift_certificate
+                    supported_instruments: []
+                    stored_instruments: []
+                  - id: bigcommerce.store_credit
+                    name: Store Credit
+                    test_mode: false
+                    type: store_credit
+                    supported_instruments: []
+                    stored_instruments: []
```
Comment (bc-harika) [inline reference/payment_processing.yml:940]: Can we add these before `stripe`, as they are sorted alphabetically in the actual API response.

**Review by bc-harika — COMMENTED** [review]
Could you please share the link to the public API doc this PR is updating. It is bit confusing as we are updating multiple docs.

**Comment by traci707** [comment]
@bc-harika Here is the public API spec, [https://developer.bigcommerce.com/api-reference/YXBpOjIzMTM3NTU-payment-processing](https://developer.bigcommerce.com/api-reference/YXBpOjIzMTM3NTU-payment-processing)

**Review by bc-tgomez — DISMISSED** [review]
Looks good to me. 😄

**Review by markcmurphy — APPROVED** [review]

**Pull request merged**